src/site/topic8.rst

```diff
@@ -274,8 +274,83 @@ Introduction Errors for Fun
    :align: center
 
 
+Playing
+=======
+
+* We can use the same code we used for the ``ArrayStack`` to play with the ``LinkedStack``
+* We only need to make one change
+    * ``ArrayStack`` -> ``LinkedStack``
+
+.. code-block:: java
+    :linenos:
+    
+    // Create an ArrayStack
+    Stack<Integer> myStack = new LinkedStack<Integer>();
+
+    // Check stack is empty
```

Comment:
Again semi related to #46, but you could include these as actual main classes in the repo; like, actual java source files.

Reply:
Grand idea. I'll create a suite of these.